### PR TITLE
Update `CONTRIBUTING.md` to include information on Git tags

### PR DESCRIPTION
**Pull request by danoc**
@tomgenoni – Can you review?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,11 +1,8 @@
 # Contributing to LEGO
 
 1. Branch off of master: `git checkout -b username/branch-name`.
-2. Make your changes, commit your code, and open a pull request. Add at least
-  one reviewer.
-3. Add your change to the "Unreleased" section of the `CHANGELOG.md` under one
-  of the following sub-sections:
-
+2. Make your changes, commit your code, and open a pull request. Add at least one reviewer.
```
Comment (tomgenoni) [inline CONTRIBUTING.md]: Can you add 'against master' or whatever the word is.

Comment (danoc) [inline CONTRIBUTING.md]: @tomgenoni – Done.

**Comment by tomgenoni** [comment]
LGTM

**Comment by danoc** [comment]
Merged.

**Pull request closed without merging**